find.go

```diff
@@ -53,6 +53,88 @@ func (s *server) findMultihashSubtree(w http.ResponseWriter, r *http.Request) {
 	}
 }
 
+func (s *server) findMetadataSubtree(w http.ResponseWriter, r *http.Request) {
+	discardBody(r)
+	if r.Method != http.MethodGet {
+		http.Error(w, "", http.StatusNotFound)
+		return
+	}
+
+	ctx := r.Context()
+	method := r.Method
+	req := r.URL
+
+	sg := &scatterGather[*url.URL, []byte]{
+		targets: s.servers,
+		tcb:     s.serverCallers,
+		maxWait: config.Server.ResultMaxWait,
+	}
+
+	var count int32
```

Comment:
`count` seem to be incremented but never used. Is it needed?

Reply:
Good spot, side effect from removing metrics. Removed.